src/utils/clickup.ts

```diff
@@ -241,35 +207,56 @@ export class Clickup implements ReleaseStrategy {
         };
     }
 
-    async createRegressionTestingIssue(version: Issue): Promise<Issue> {
-        const title = `${PLATFORM} Regression Tag - ${TAG}`;
-        const tasks = await this.fetchIssues(LIST_ID);
-        const has_regression_testing_card = tasks.some(task => task.title === title);
-        let regression_testing_card: Issue;
+    async fetchTasksFromReleaseTagTask(release_tag_task_url: string): Promise<Issue[]> {
+        const issues: Issue[] = [];
 
-        if (!has_regression_testing_card) {
-            // If regression testing card doesn't exist, create it
-            logger.log(`Creating regression testing card with title ${title}...`, 'loading');
-            const task = await this.http.post<Task>(`list/${LIST_ID}/taskTemplate/${REGRESSION_TESTING_TEMPLATE_ID}`, {
-                name: title,
-            });
-            await this.updateIssue(task.id, {
-                status: 'Pending - QA',
-            });
-            regression_testing_card = {
-                id: task.id,
-                title: task.name,
-                description: task.description,
-                status: 'Pending - QA',
-            };
+        const { task_id, team_id } = this.getTaskIdAndTeamIdFromUrl(release_tag_task_url);
+
+        const task = await this.http.get<Task>(`task/${task_id}?team_id=${team_id}&custom_task_ids=true`);
+        this.regession_task = task;
+        const { custom_fields } = task;
+        const task_ids = this.getTasksIdsFromCustomFields(custom_fields);
+        for (const task_id of task_ids) {
+            const task = await this.fetchIssue(task_id);
+            issues.push(task);
+        }
+
+        return issues;
+    }
+    getTaskIdAndTeamIdFromUrl(url: string) {
+        const pattern = /https:\/\/app\.clickup\.com\/t\/([\w-]*)\/*([\w-]*)/;
+        const matches = pattern.exec(url);
+        const ids = matches?.slice(matches?.length - 2) ?? ['', ''];
+        let task_id = '';
+        let team_id = '';
+
+        if (ids.length > 0 && ids[ids.length - 1]) {
+            [team_id, task_id] = ids;
         } else {
-            logger.log(`Regression testing card has already been created.`, 'success');
-            regression_testing_card = tasks.find(task => task.title === title) as Issue;
+            [task_id] = ids;
+        }
+
+        return { task_id, team_id };
+    }
+
+    getTasksIdsFromCustomFields(custom_fields?: CustomField[]): string[] {
+        const taskIds: string[] = [];
+        for (const custom_field of custom_fields ?? []) {
```

Comment:
```suggestion
        for (const field of custom_fields) {
```
More readable this way. And for the default value, rather than a fallback here we should set it as a default value in the function argument